backend/push.ts

```
@@ -0,0 +1,56 @@
+import { SessionData } from 'express-session'
+import mongoose from 'mongoose'
+import webpush from 'web-push'
+import { User } from './../common/types.js'
+import { sessionStore } from './app.js'
+
+webpush.setVapidDetails(process.env.VITE_FRONTEND_URL, process.env.VITE_PUBLIC_VAPID_KEY, process.env.VITE_PRIVATE_VAPID_KEY)
+
+export async function sendPushNotification(title: String, body: String, users: User[], url: string) {
```

Biome

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)